app/components/svg_component.rb

```diff
@@ -6,6 +6,7 @@ class SvgComponent < ApplicationComponent
     gear: :gear,
     map: :map,
     receipt_percent: :receipt_percent,
+    letter: :receipt_percent,
```

Comment:
This should probably be a better SVG than the receipt one...

Reply:
the `ICON_MAPPINGS` was named `letter` so I replaced it with letter icon in 85514fdc1b22a7d4e3713f052521fa2be6e7d88d

![letter icon](https://github.com/zinc-collective/convene/assets/16140873/c7788bc6-b619-4267-bb8b-e38cda3fc358)

But, since the icon is applied to a notification menu, I felt a bell icon worked better, thus I added it in a3e90d9337cb107af31120c16c4064fc633452e9. 
![bell icon](https://github.com/zinc-collective/convene/assets/16140873/bd0ef7bb-8568-48f2-ac39-75f6ae2b564f)


Please choose which on you want or suggest other icon suggestions.

Reply:
Oh I see now, it's notification via email. Hm maybe the open_letter icon works better than.

Maybe we should also change the `Order Notifications` text to a clearer `Order Email Notifications` 🤔

Reply:
Well, it's notifications via email... _for now_; but if you check the [`Order` `Notifications` and History](https://github.com/zinc-collective/convene/issues/1331) we do have plans to include at the very least Square Point of Sale; and maybe even SMS!

Reply:
Also, I love the bell!